### PR TITLE
Add JetBrains annotations

### PR DESCRIPTION
**Pull request by TheChilliPL**

### CODE REVIEW

File: AudioSensei/AvaloniaExtensions.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace AudioSensei
 {
     public static class AvaloniaExtensions
     {
-        public static void SetForegroundWindow(this Window window)
+        public static void SetForegroundWindow([JetBrains.Annotations.CanBeNull] this Window window)
```
Comment (KernelErr0r) [inline AudioSensei/AvaloniaExtensions.cs:12]: Why is there a namespace?

File: AudioSensei/Program.cs [hunk 2]
```diff
@@ -26,7 +27,7 @@ namespace AudioSensei
 {
     class Program
     {
-        public static event Action Exit;
+        [NotNull] public static event Action Exit;
```
Comment (KernelErr0r) [inline AudioSensei/Program.cs:30]: It shouldn't be in the same line

File: AudioSensei/AvaloniaExtensions.cs (hunk 1 above)
Comment (TheChilliPL) [reply AudioSensei/AvaloniaExtensions.cs:12]: I have literally no idea why.
It should obviously be:
```suggestion
        public static void SetForegroundWindow([CanBeNull] this Window window)
```

File: AudioSensei/Program.cs (hunk 2 above)
Comment (MichalPetryka) [reply AudioSensei/Program.cs:30]: This isn't even correct

Comment (TheChilliPL) [reply AudioSensei/Program.cs:30]: I wasn't sure what formatting conventions you use
```suggestion
        [NotNull]
        public static event Action Exit;
```

Comment (MichalPetryka) [reply AudioSensei/Program.cs:30]: But it can be null though

File: AudioSensei/ViewModels/MainWindowViewModel.cs [hunk 3]
```diff
@@ -741,11 +750,13 @@ private void CancelPlaylistCreation()
             PlaylistDescription = "";
         }
 
+        [NotNull]
```
Comment (KernelErr0r) [inline AudioSensei/ViewModels/MainWindowViewModel.cs:753]: These void things are weird, we should make sure it won't be a null, right? right? right?

File: AudioSensei/Program.cs (hunk 2 above)
Comment (TheChilliPL) [reply AudioSensei/Program.cs:30]:
```suggestion
        [CanBeNull]
        public static event Action Exit;
```

File: AudioSensei/ViewModels/MainWindowViewModel.cs (hunk 3 above)
Comment (TheChilliPL) [reply AudioSensei/ViewModels/MainWindowViewModel.cs:753]: No idea how did that happen
```suggestion
```

File: AudioSensei/ViewModels/MainWindowViewModel.cs [hunk 4]
```diff
@@ -380,7 +381,8 @@ private async void ProcessTcp()
             Log.Information("Stopping listening to playback requests");
         }
 
-        private async Task ProcessClient(TcpClient client)
+        [NotNull]
+        private async Task ProcessClient([NotNull] TcpClient client)
```
Comment (KernelErr0r) [inline AudioSensei/ViewModels/MainWindowViewModel.cs:385]: I'm not even sure if that makes sense, probably not.

File: AudioSensei/WebHelper.cs [hunk 5]
```diff
@@ -2,11 +2,13 @@
 using System.Net;
 using System.Net.Http;
 using System.Runtime.InteropServices;
+using DynamicData.Annotations;
```
Comment (KernelErr0r) [inline AudioSensei/WebHelper.cs:5]: Are you sure?

File: AudioSensei/ViewModels/MainWindowViewModel.cs (hunk 4 above)
Comment (KernelErr0r) [reply AudioSensei/ViewModels/MainWindowViewModel.cs:385]: I think it's unnecessary in that case

File: AudioSensei/WebHelper.cs (hunk 5 above)
Comment (TheChilliPL) [reply AudioSensei/WebHelper.cs:5]: There are 3 different [NotNull] and [CanBeNull] annotations there, in three different namespaces.
Usings were added by Rider, after I explicitly clicked which one, might've made a small mistake
```suggestion
using JetBrains.Annotations;
```

**Pull request merged**